<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>157</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>8</y>
      <width>132</width>
      <height>125</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pb_init">
       <property name="text">
        <string>初始化模型</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_choice">
       <property name="text">
        <string>选择图像</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_predict">
       <property name="text">
        <string>开始优化</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>  状态 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_now">
         <property name="text">
          <string>请初始化模型</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
